CorsixTH/Lua/app.lua

```diff
@@ -716,7 +716,7 @@ end
 --! Loads the specified level. If a string is passed it looks for the file with the same name
 -- in the "Levels" folder of CorsixTH, if it is a number it tries to load that level from
 -- the original game.
-function App:_loadLevel(level, difficulty, level_name, level_file, level_intro, map_editor)
+function App:_loadLevel(level, difficulty, level_name, level_file, level_intro, map_editor, campaign_info)
```

Comment:
Can we add some parameter documentation.

IMO the number of parameters here is getting silly.

Reply:
I agree that number of parameters here is getting silly. 💯

I have added campaign_info parameter documentation here.

About other params I can't tell more that their names says.

Reply:
The doc will be a mirror of `function:App:loadLevel(...)` bar the error paramter. I'm not sure it requires documenting twice but c'est la vie :)